src/determinestrategy.jl

```diff
@@ -239,110 +239,110 @@ function determine_unroll_factor(
     roundpow2(max(1, round(Int, latency / (recip_throughput * num_reductions) ) ))
 end
 
-function tile_cost(X, U, T, UL, TL)
-    Tfactor = (num_iterations(TL, T)/TL)
-    Ufactor = (num_iterations(UL, U)/UL)
-    # X[1]*Tfactor*Ufactor + X[4] + X[2] * Tfactor + X[3] * Ufactor
-    X[1] + X[4] + X[2] * Tfactor + X[3] * Ufactor
+function unroll_cost(X, u₁, u₂, u₁L, u₂L)
+    u₂factor = (num_iterations(u₂L, u₂)/u₂L)
+    u₁factor = (num_iterations(u₁L, u₁)/u₁L)
```

Comment:
I just realized `num_iterations` should be replaced with `cld`.

Reply:
Yes, it would compile faster, though I bet that's not a big item.